Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/AssetPage.tsx

```diff
@@ -560,67 +560,67 @@ export default function AssetPage(props: IProps) {
-                                            setNewEditAsset(asset);
-
-                                            props.UpdateChannels(channels);
-                                        }} value={newEditAsset.Channels.filter(ch => ch.ConnectionPriority == 1).map(a => a.ID.toString())}>
-                                            {
-                                                props.Channels.map((channel, index) => <option key={index} value={channel.ID} hidden={channel.Asset != newEditAsset.AssetKey && channel.Asset.length > 0}>{channel.Name + ' - ' + channel.Description}</option>)
-                                            }
-                                        </select>
-                                    </div>
-                                    <div className="col-4">
-                                        <label>Associated Channels Tertiary Side</label>
-                                        <select multiple style={{ height: innerHeight - 430, width: '100%' }} onChange={(evt) => {
-                                            let asset = _.clone(newEditAsset as OpenXDA.Types.Asset);
-                                            const selectedID = ($(evt.target).val() as Array<string>).map(a => parseInt(a));
-                                            let channels = _.clone(props.Channels);
-                                            channels = channels.map(ch => ({ ...ch, ConnectionPriority: (selectedID.indexOf(ch.ID) == -1 ? ch.ConnectionPriority : 2) }));
-
-                                            asset.Channels = asset.Channels.filter(ch => ch.ConnectionPriority == 0 || ch.ConnectionPriority == 1).concat(selectedID.map(a => channels.find(ch => ch.ID == a)));
-                                            setNewEditAsset(asset);
-
-                                            props.UpdateChannels(channels);
-                                        }} value={newEditAsset.Channels.filter(ch => ch.ConnectionPriority == 2).map(a => a.ID.toString())}>
-                                            {
-                                                props.Channels.map((channel, index) => <option key={index} value={channel.ID} hidden={channel.Asset != newEditAsset.AssetKey && channel.Asset.length > 0}>{channel.Name + ' - ' + channel.Description}</option>)
-                                            }
-                                        </select>
-                                    </div>
-                                </div>
-                            </div> : null}
-                            </div>
+                                {newEditAsset.AssetType === 'Transformer' ?
+                                    <div className="col-12">
+                                        <div className="row justify-content-center">
+                                            <div className="col-3 mr-5">
```

Comment:
You have 3 `col-3` here. That does not add up to take up all the available space. The total space is `12` so 3 `col-4` would do it.